idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/markers/ProofMarkers.kt

```diff
@@ -105,9 +106,4 @@ inline fun <reified A : KtDeclaration> IdeMetaPlugin.proofLineMarkers(icon: Icon
   )
 
 fun CoercionProof.coercionMessage(): String =
```

Comment:
Maybe we can rename this to make it a bit more clear it's a shared one to be embed into different specific copies.
```suggestion
fun CoercionProof.coercionDefaultMessage(): String =
```